content/markdown-converter.js

```diff
@@ -0,0 +1,23 @@
+const unified = require("unified");
+const parse = require("remark-parse");
+const stringify = require("rehype-stringify");
+const remark2rehype = require("remark-rehype");
+const raw = require("rehype-raw");
+
+/**
+ * Converts Markdown -> HTML using unified.
+ * Using `raw` enables us to process HTML embedded in the Markdown.
+ */
+function markdownToHTML(md) {
+  return unified()
```

Comment:
I came for this :-D
Haven't heard of `unified`.
(I'd have tried [turndown](https://www.npmjs.com/package/turndown) based on prior experience with it), but good to know alternatives!

Reply:
Turndown is promising. I used it to make some conversions but it'll need a lot more careful thought (and time). 

My dream is that writers can gently pick one document at a time and press a button on the Toolbar that generates a rough draft (probably with Turndown) and then the writer manually makes all the necessary fixes to get it to look stunning. But we'll have to be careful about the scary things that aren't easy to convert, such as note cards, live samples, `<dl>` etc. The list is long.

Reply:
Then, I'd recommend to decide (and document publicly) which Markdown flavour you want to roll with.
GFM would be natural choice. Commonmark documented other ones:
https://github.com/commonmark/commonmark-spec/wiki/Markdown-Flavors

Depending on this, you'll need to pick the tooling.